resource-aggregate/cqrs/eventstore/mongodb/eventstore.go

```diff
@@ -26,20 +24,40 @@ import (
 
 const eventCName = "events"
 
-const aggregateIDKey = "aggregateid"
-const aggregateIDStrKey = "aggregateidstr"
-const idKey = "_id"
+// Event
 const versionKey = "version"
 const dataKey = "data"
 const eventTypeKey = "eventtype"
+const isSnapshotKey = "issnapshot"
```

Comment:
why are you not using camelCase?

Reply:
mongo all keys string changed to lower so we want to match it.